keps/2724-topology-aware-scheduling/README.md

```diff
@@ -1153,6 +1154,43 @@ to ungate a Pod. The expectation is fulfilled if the Pod is observed as ungated
 or the ungating request fails. We hold ungating if there are pending ungatings
 within the PodSet.
 
+### Balanced placement
```

Comment:
Earlier in this KEP, [this](https://github.com/kubernetes-sigs/kueue/blob/15ea4e843157cb86b443ad8ff0f5ebeed9327ef2/keps/2724-topology-aware-scheduling/README.md?plain=1#L1034-L1037) looks like a list of supported algorithms. \
How about mentioning `Balanced` also there? (Just briefly, with a link to this section).